package.json

```
@@ -10,6 +10,7 @@
     "node": "^12.16.3"
   },
   "scripts": {
+    "storybook-axe": "yarn ts-node storybook/bin/check-storybook-accessibility",
```

Meganit, take it or leave it
```suggestion
    "storybook:axe": "yarn ts-node storybook/bin/check-storybook-accessibility",
```